modules/tls-profiles-kubelet-configuring.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubelet-configuring_{context}"]
+= Configuring TLS profiles for the Kubelet
+
+To configure a TLS security profile for the kubelet when it is acting as an HTTP server, create a `KubeletConfig` custom resource (CR) to specify a predefined or custom TLS profile for specific nodes. If a TLS profile is not configured, the default TLS profile is `Intermediate`.
+
+.Sample `KubeletConfig` CR that configures the `Old` TLS security profile on worker nodes
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: KubeletConfig
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: ""
+ ...
```

Comment:
Just checking if this `...` is needed, because in the next YAML block, it ends here.

Plus it's being highlighted weirdly in red, so best to remove if it's not really necessary